### PR TITLE
dxGantt: add examples

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md [hunk 1]
```diff
@@ -24,6 +24,99 @@ Specifies whether users can select tasks in the Gantt.
...
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxGantt
+            :allow-selection="true" >
```
Comment (Kobart13) [inline api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md:69]: Did you mean
:allow-selection="false" ?

File: api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md [hunk 2]
```diff
@@ -24,6 +24,99 @@ Specifies whether users can select tasks in the Gantt.
...
+        );
+    };
+
+    export default App;
+
+##### ASP.NET MVC Controls
+
+    <!--Razor C#-->
+    @(Html.DevExtreme().Gantt()
+        .AllowSelection(true)
```
Comment (Kobart13) [inline api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md:117]: Did you mean
.AllowSelection(false) ?

File: api-reference/_hidden/dxGanttContextMenuItem/name.md [hunk 3]
```diff
@@ -8,3 +8,172 @@ type: String
...
+                        ]                        
+                    }
+                    // ...
+                ]
+            }
+            onCustomCommand: onCustomCommandClick
+            // ...
+        });
+    });
+    function onCustomCommandClick(args) {
```
Comment (Kobart13) [inline api-reference/_hidden/dxGanttContextMenuItem/name.md:45]: you have _args_ in function parameters, but _e.name_ is used

File: api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md:69]: yes, fixed

File: api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md (hunk 2 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Components/dxGantt/1 Configuration/allowSelection.md:117]: yes, fixed

File: api-reference/_hidden/dxGanttContextMenuItem/name.md (hunk 3 above)
Comment (AnnaKondratova) [reply api-reference/_hidden/dxGanttContextMenuItem/name.md:45]: fixed

**Review by Kobart13 — APPROVED** [review]

**Pull request merged**